lib/ModelMixins/DiffableMixin.ts

```diff
@@ -56,6 +55,24 @@ class DiffStratum extends LoadableStratum(DiffableTraits) {
   get disableDateTimeSelector() {
     return this.catalogItem.isShowingDiff;
   }
+
+  @computed
+  get disableExport() {
+    // disable export if showing diff
+    // currently there is no way to generate export for the difference layer as
+    // it requires 2 time parameters which is not supported in standard WCS
+    const disableExport = this.catalogItem.isShowingDiff;
```

Comment:
Is the intermediate variable required for some JavaScript reason, or could the method directly  `return this.catalogItem.isShowingDiff` just as well?

Reply:
Fixed. I previously wrote it like `this.catalogItem.isShowingDiff ? true : false` because it reads better, but the linter doesn't like it.